rdf-delta-cmds/src/main/java/delta/server/ServerBuildLib.java

```diff
@@ -137,8 +138,15 @@ private static LocalServerConfig serverConfigZookeeper(DeltaServerConfig config)
 
         if ( config.zkConnectionString == null )
             throw new DeltaConfigException("No connection string for ZooKeeper");
+
+        Properties props = null;
+        if ( config.zkRootDirName != null ) {
+            props = new Properties();
+            props.setProperty(ZkConst.prRootDirName, config.zkRootDirName);
+        }
+
         // If zk+S3, there isn't a provider name set yet.
-        LocalServerConfig localServerConfig = LocalServers.configZk(config.zkConnectionString, config.jettyConf);
+        LocalServerConfig localServerConfig = LocalServers.configZk(config.zkConnectionString, config.jettyConf, props);
```

Comment:
Does this have to be done by Properties and not a String zkRootDir argument? This is "configZk" so it can be ZK- specific.